CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ The sections should follow the order `Packaging`, `Added`, `Changed`, `Fixed` an
 
 ## Added
 - Option to hide ignored addons.
+- Load API catalogs asynchronously.
```

Comment:
Let's move this to `## Changed`or `## Fixed`.

Reply:
Done